### ISSUE TITLE
Suggestion for revamping documentation structure

### ISSUE THREAD

**Issue by dennisnewel** [jenkins-x/jx#5157]
As part of the community survey CloudBees did a short while ago, it was clear that a big issue for most people was that the documentation on https://jenkins-x.io is confusing and lacking in various areas.

I’d like to propose a new structure and a way for how we can arrange the existing documentation in the new structure. 
Once in place, I want to go through and identify the major gaps that we have and start working on closing those.

The proposed structure and my suggested mapping is in this google sheets: https://docs.google.com/spreadsheets/d/13z9qBkrbNGc3MrmyAd-Plr8r8pRmuHeQEnvk244vhD4/edit?usp=sharing (which anyone can comment on).

TL;DR; of the spreadsheet is a structure focused more on either using or managing Jenkins X, with dedicated structures for Getting Started, Contributing, etc. 
Most of the content would just be moved over, with a few pages being split between structures depending on whether it’s for using or managing Jenkins X

**Comment by dennisnewel** [comment]
this work is now happening in the `site-restructure` branch https://github.com/jenkins-x/jx-docs/tree/site-restructure.
It's a fresh setup of Docsy with the old site in a `/old` folder, where we're in the progress of moving the old content into the new structure

**Comment by babadofar** [comment]
Thank you this re-organization is very welcome. Also, there seem to be some missing pieces. For instance, gitops, it is described how you can install it, but the gitops operations and commands are as far as I can see not documented. 
The documentation should be clearer on what concerns the old static Jenkins  kind of install, and the new Serverless style.

**Comment by dennisnewel** [comment]
Good points @babadofar! would you mind raising a couple of issues on that in the `jx` repo and label them 'area/docs'? i'm trying to keep track of all the feedback on https://github.com/orgs/jenkins-x/projects/5 and will add yours as well. Once the re-org is in place, the plan is to go through and update everything and add what's missing :)

**Comment by tiger74s** [comment]
For most of current Jenkins users, we might want to know a little bit of history of Jenkins X.  In additional "where to go next", I would like to see where it came from, why do we even need JX.  
Jenkins X uses another open source project Tekton.  Jenkins users might also want to know why Jenkins X uses Tekton to replace Jenkins as the pipeline engine.

**Comment by dennisnewel** [comment]
given jenkins-x/jx-docs#2041 and jenkins-x/jx-docs#2039 have been created to track that feedback, i'm going to close this issue as the revamping has been completed
